Converter/igtlioTrackingDataConverter.cxx

```diff
@@ -20,6 +20,13 @@ int igtlioTrackingDataConverter::fromIGTL(igtl::MessageBase::Pointer source,
     bool checkCRC,
     igtl::MessageBase::MetaDataMap& outMetaInfo)
 {
+
+  // Process an RTS_TDATA sub-type message
+  if (strncmp(source->GetDeviceType(), "RTS_", 4) == 0)
```

Comment:
Should this be: "source->GetMessageType()"?

Reply:
Good idea. Both are acceptable in the OpenIGTLink library but "device" might be confusing in the context of OpenIGTLinkIO. I'll fix it.